Doppler.Sap/Models/SapIncomingPaymentModel.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using System.Collections.Generic;
+
+namespace Doppler.Sap.Models
+{
+    public class SapIncomingPaymentModel
+    {
+        public string DocType { get; set; }
+        public string DocDate { get; set; } = DateTime.UtcNow.ToString("yyyy-MM-dd");
+        public string CardCode { get; set; }
+        public string DocCurrency { get; set; }
+        public string TransferAccount { get; set; }
+        public double TransferSum { get; set; }
+        public string TransferDate { get; set; } = DateTime.UtcNow.ToString("yyyy-MM-dd");
+        public string TransferReference { get; set; }
+        public string JournalRemarks { get; set; }
+        public string TaxDate { get; set; } = DateTime.UtcNow.ToString("yyyy-MM-dd");
```

Comment:
Is it fine using UTC? take into account that depending in the timezone the day could change

Reply:
yes, you are right. @fgchaio / @swidzinskiMS , do you know why was utc used for dates invoice dates?:
```
public string DocDate { get; set; } = DateTime.UtcNow.ToString("yyyy-MM-dd");
public string DocDueDate { get; set; } = DateTime.UtcNow.ToString("yyyy-MM-dd");
public string TaxDate { get; set; } = DateTime.UtcNow.ToString("yyyy-MM-dd");
```

I used it to be consistent with invoices.

Reply:
I guess it was like that because we don't have the client's time zone in sap.

Reply:
But, should we not use the Argentinian date?

Reply:
I mean, you generate an invoice at `2020-12-03 22:10:00 ARG` (or, the same, `2020-12-04 01:10:00 UTC`), what is the invoice date `2020-12-03` or `2020-12-04`?

Reply:
I don't know.
Does it make any difference for the client? If it does, we should take into account the time zone.

Reply:
Maybe we can check it with Lucas. I will check in our channel.

Reply:
@jhoffman-ms what do you think about avoiding these default values and generate the values directly in `MapSapIncomingPayment`? I think that in the future you will have better tools there.

Another thing, what do you think about using the complete UTC datetime in place of only the date? it will allow you to have more information and postpone the decision about what date to show for the future.

Reply:
Yes, I will remove the default values from the ```SapIncomingPaymentModel``` class and I will add these values in the ```MapSapIncomingPayment``` method.

Reply:
I moved the default values into the ```MapSapIncomingPayment``` method.

Reply:
In our daily call we decided that for the moment is correct use UtcNow. Maybe in the future we will create a new task to change it.